submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css

```diff
@@ -0,0 +1,366 @@
+* {
+    box-sizing: border-box;
+}
+
+html,body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: auto;
+    overflow-y: auto;
+}
+
+
+body {
+    font-family: 'Montserrat', 'Raleway', sans-serif;
+}
+
+.header {
+    background-color: rgba(000, 255, 000, 0.4);
+}
+
+nav {
+    display: flex;
+    margin: 0 10px;
+}
+
+
+.left-tab {
+    width: 50%;
+    float: left;
+    margin: 0 20px;
+}
+
+.right-sidetab {
+    width: 50%;
+    text-align: end;
+    margin: auto;
+
+}
+.right-sidetab>a {
+    text-decoration: none;
+    color: black;
+    padding: 0 30px;
+    
+}
+
+main {
+    display: flex;
+}
+
+.left-main {
+    width: 50%;
+    display: block;
+    margin: 25px;
+    text-align: center;
+}
+
+.border-left {
+    padding: 30px 50px;
+    white-space: nowrap;
+    margin: auto;
+}
+
+.phone-sign>p:nth-child(2) {
+    font-weight: bold;
+}
+
+.phone-sign>p>a {
+    text-decoration: none;
+    color: black;
+}
+
+.open-email>img {
+    padding-top: 10px;
+}
+
+.open-email>p {
+    font-weight: bold;
+}
+
+.open-email>p>a {
+    text-decoration: none;
+    color: black;
+    font-weight: normal;
+}
+
+.social-media {
+    padding-top: 20px;
+}
+
+.social-media>p {
+    font-weight: bold;
+}
+
+.right-main {
+    border: 1px solid black;
+    width: 50%;
+    margin: 25px;
+}
+
+.border-right {
+    padding: 30px;
+}
+
+.message>img {
+    display: block;
+    margin: auto;
+    padding-bottom: 10px;
+}
+
+.message>p {
+    text-align: center;
+    margin: auto;
+    padding-bottom: 10px;
+}
+
+.form>input.full-name,.form>input.email,.form>input.subject {
+    width: 100%;
+    height: 0;
```

Comment:
You won't be able to type into your input box if you are setting the height to 0. I will advise that you remove this.